### PR TITLE
Fixs after release

### PR DESCRIPTION
**Pull request by rewin123**
1. Fixed mouse picking bug, when can be selected entity without PrefabMarker
2. Change play button stroke from light blue to dark gray
3. Added button in top menu, which allow fast add to scene glb/gltf/scn.ron based entities with file dialog

### CODE REVIEW

File: crates/editor_ui/src/hierarchy.rs [hunk 1]
```diff
@@ -150,23 +154,30 @@ fn draw_entity<F: ReadOnlyWorldQuery>(
...
-                    entity,
-                    changes,
-                    clone_events,
-                    selected,
-                    parent,
-                );
-            });
+            if is_auto_child {
+                response.context_menu(|ui| {
+                    ui.label("Its auto child prefab entity of solid bevy scene. Can't be reparented/deleted. Try \"Unpack gltf as prefab\" for that.");
```
Comment (naomijub) [inline crates/editor_ui/src/hierarchy.rs:166]: The phrasing got a little confusing, Did you mean something like `"Concrete Bevy scene auto child cannot be reparented or deleted. Try \"Unpack gltf as prefab\" for that.".` Also, this feels more like an error toast than a label

Comment (naomijub) [reply crates/editor_ui/src/hierarchy.rs:166]: Having it as an event will be easier to test as well and will be registered in the event window (Top right corner).
`mut toast: EventWriter<ToastMessage>`

`toast.send(ToastMessage::new(Concrete Bevy scene auto child cannot be reparented or deleted. Try \"Unpack gltf as prefab\" for that.", ToastKind::Error)`

File: crates/editor_ui/src/hierarchy.rs [hunk 2]
```diff
@@ -200,20 +211,32 @@ fn draw_entity<F: ReadOnlyWorldQuery>(
...
-                entity,
-                changes,
-                clone_events,
-                selected,
-                parent,
-            );
-        });
+        if is_auto_child {
+            selectable.context_menu(|ui| {
+                ui.label("Its auto child prefab entity of solid bevy scene. Can't be reparented/deleted. Try \"Unpack gltf as prefab\" for that.");
```
Comment (naomijub) [inline crates/editor_ui/src/hierarchy.rs:225]: Same as before

File: crates/editor_ui/src/menu_toolbars.rs [hunk 3]
```diff
@@ -462,14 +464,65 @@ pub fn top_menu(
...
+
                 let width = ui.available_width();
                 let distance = width / 2. - 40.;
                 ui.add_space(distance);
                 let play_button = egui::Button::new(to_richtext("▶", &sizing.icon))
                     .fill(SPECIAL_BG_COLOR)
                     .stroke(Stroke {
                         width: 1.,
-                        color: SELECTED_ITEM_COLOR,
+                        color: egui::Color32::DARK_GRAY,
```
Comment (naomijub) [inline crates/editor_ui/src/menu_toolbars.rs:525]: There was a cool reason for it to be dark blue:
1. More professional engines, like Unreal and Cryengine, use a nice color schema to differentiate important buttons. The play button is usually one of those
2. For color blind people, the default color used in those cases, green, is quite hard to see, being almost black, while this tone of blue is easily seen. 
3. Dark gray might be confusing with other lower level icons

File: crates/editor_ui/src/hierarchy.rs (hunk 1 above)
Comment (rewin123) [reply crates/editor_ui/src/hierarchy.rs:166]: Its not error xDD Its just reminder that it is not "full" prefab entity and only support part of functionality

File: crates/editor_ui/src/menu_toolbars.rs (hunk 3 above)
Comment (rewin123) [reply crates/editor_ui/src/menu_toolbars.rs:525]: I really tried 20 seconds to unselect button. Select color its not good. Can we select another for it then?

Comment (naomijub) [reply crates/editor_ui/src/menu_toolbars.rs:525]: Sure we can use another color, I just didn't understand the problem, can you rephrase?

File: crates/editor_ui/src/hierarchy.rs (hunk 1 above)
Comment (naomijub) [reply crates/editor_ui/src/hierarchy.rs:166]: We could probably add an warning label to that. Can you create an example where this happens so I can discuss with the UX?

File: crates/editor_ui/src/menu_toolbars.rs (hunk 3 above)
Comment (rewin123) [reply crates/editor_ui/src/menu_toolbars.rs:525]: I tried to deselect the Play button for about 20 seconds xDD

File: crates/editor_ui/src/hierarchy.rs (hunk 1 above)
Comment (rewin123) [reply crates/editor_ui/src/hierarchy.rs:166]: This message is displayed instead of the context menu when right-clicking an entity in the hierarchy if the entity is an incomplete prefab

File: crates/editor_ui/src/menu_toolbars.rs (hunk 3 above)
Comment (naomijub) [reply crates/editor_ui/src/menu_toolbars.rs:525]: https://github.com/rewin123/space_editor/pull/238

File: crates/editor_ui/src/hierarchy.rs (hunk 1 above)
Comment (naomijub) [reply crates/editor_ui/src/hierarchy.rs:166]: I would just make the text clearer:

"Concrete Bevy scene cannot be reparented or deleted. Try \"Unpack gltf as prefab\" for that." 

And add a red label with color: ERROR_COLOR

Comment (naomijub) [reply crates/editor_ui/src/hierarchy.rs:166]: Solved https://github.com/rewin123/space_editor/pull/239

File: crates/editor_ui/src/hierarchy.rs (hunk 2 above)
Comment (naomijub) [reply crates/editor_ui/src/hierarchy.rs:225]: Solved https://github.com/rewin123/space_editor/pull/239

File: crates/editor_ui/src/hierarchy.rs (hunk 1 above)
Comment (naomijub) [reply crates/editor_ui/src/hierarchy.rs:166]: Captura de Tela 2024-03-05 às 7 49 31 PM

File: crates/editor_ui/src/menu_toolbars.rs (hunk 3 above)
Comment (naomijub) [reply crates/editor_ui/src/menu_toolbars.rs:525]: Captura de Tela 2024-03-05 às 7 11 43 PM

Uses unreal style
